### PR TITLE
Revert "Fix telemetry unicode errors (#1937)"

### PR DESCRIPTION
**Pull request by pgombar**
## Description

Revert #1937
This reverts commit 6d1020325d46bd4a9ba3e467021d2926cdf6f173.

---

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and Travis.CI is passing.

### Quality of Code and Contribution Guidelines
- [ ] I have read the [contribution guidelines](https://github.com/Azure/WALinuxAgent/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: tests/common/test_event.py [hunk 1]
```diff
@@ -412,19 +411,17 @@ def test_collect_events_should_ignore_invalid_event_files(self):
...
 
-            self.assertEqual(3, total_dropped_count, "Total dropped events dont match")
-            self.assertIn(WALAEventOperation.CollectEventErrors, invalid_events,
-                          "{0} errors not reported".format(WALAEventOperation.CollectEventErrors))
-            self.assertIn(WALAEventOperation.CollectEventUnicodeErrors, invalid_events,
-                          "{0} errors not reported".format(WALAEventOperation.CollectEventUnicodeErrors))
+            def assert_invalid_file_was_reported(filename):
+                self.assertIn(filename, invalid_events, '{0} was not reported as an invalid event file'.format(filename))
+
+            assert_invalid_file_was_reported("custom_script_utf-16.tld")
```
Comment (pgombar) [inline tests/common/test_event.py:422]: This is failing, saying this file is not invalid

File: tests/common/test_event.py [hunk 2]
```diff
@@ -412,19 +411,17 @@ def test_collect_events_should_ignore_invalid_event_files(self):
...
+            assert_invalid_file_was_reported("custom_script_invalid_json.tld")
```
Comment (pgombar) [inline tests/common/test_event.py:423]: This is failing, saying this file is not invalid

File: azurelinuxagent/common/utils/textutil.py [hunk 3]
```diff
@@ -398,35 +396,3 @@ def format_memory_value(unit, value):
         raise TypeError('Value must be convertible to a float')
 
     return int(value * units[unit])
-
-def str_to_encoded_ustr(s, encoding='utf-8'): # pylint: disable=C0103
```
Comment (larohra) [inline azurelinuxagent/common/utils/textutil.py:402]: I remember I was using this function outside this PR too, so just keep an eye out to make sure all references of this are removed

File: tests/common/test_event.py (hunk 1 above)
Comment (larohra) [reply tests/common/test_event.py:422]: A quick question, is it failing after the revert or before the revert?

**Review by larohra — APPROVED** [review]
Minor comments but in general LGTM 
(as a final sanity check, you could also run DCR on this branch just to be doubly sure)

File: tests/common/test_event.py (hunk 1 above)
Comment (pgombar) [reply tests/common/test_event.py:422]: After the revert, same for below

**Comment by pgombar** [comment]
`git revert` produces too many conflicts and I made mistakes when resolving them. I'm taking a manual revert approach, closing this PR in favour of #2070

**Pull request closed without merging**